docs/build/getting-started/networks/overview.md

```diff
@@ -1,5 +1,148 @@
-# Networks overview
+---
+description: 'As Shimmer is an innovation network, you will always find more features like NFTs, native tokens, and smart contracts
+than on the IOTA network at the cost of some stability.'
+image: /img/shimmer-wiki.png
+---
 
-Short explanation as to why there are different networks and their purpose.
+# Layer 1 Networks in the IOTA Ecosystem
 
-Table comparing networks similar to https://wiki.iota.org/shimmer/develop/explanations/what-is-shimmer/layer-1-landscape/
+:::note tl;dr;
+
+As Shimmer is an innovation network, you will always find more features like NFTs, native tokens, and smart contracts
```

Comment:
Didn't we align on staging network? I don't care, but I would at least mention both 🤔